applications/concept-catalogue/src/test/kotlin/ConceptRegistrationApplicationTest.kt

```diff
@@ -122,4 +121,281 @@ class ConceptRegistrationApplicationTests {
 
         assertFalse(sqlStore.begrepExists(begrep))
     }
+
+    @Test
+    fun testBegrepStatusValidation() {
+        val validator = { b: Begrep -> BegrepValidator.validate(b) }
+        val begrep = createBegrep()
+
+        begrep.status = Status.PUBLISERT
+        assertTrue(validator(begrep))
+
+        begrep.status = Status.GODKJENT
+        assertTrue(validator(begrep))
+
+        begrep.status = Status.UTKAST
+        assertFalse(validator(begrep))
+
+        begrep.status = null
+        assertFalse(validator(begrep))
+    }
+
+    @Test
+    fun testBegrepAnbefaltTermValidation() {
```

Comment:
Jeg antar her at anbefalt term skal ha en verdi. Et bedre navn for testen er noe slikt som testValidateAnbefaltTermNonNull eller MustHaveValue.